OpenRA.Mods.Common/UpdateRules/Rules/RenameEmitInfantryOnSell.cs

```diff
@@ -13,33 +13,32 @@
 
 namespace OpenRA.Mods.Common.UpdateRules.Rules
 {
-    public class RenameEmitInfantryOnSell : UpdateRule
-    {
-        public override string Name { get { return "EmitInfantryOnSell renamed to SpawnActorsOnSell"; } }
-        public override string Description
-        {
-            get
-            {
-                return "The EmitInfantryOnSell trait was renamed to SpawnActorsOnSell and the default\n" +
-                    "actor type to spawn was removed. Uses of the old traits and defaults are updated\n" +
-                    "to account for this.";
-            }
-        }
+	public class RenameEmitInfantryOnSell : UpdateRule
+	{
+		public override string Name { get { return "EmitInfantryOnSell renamed to SpawnActorsOnSell"; } }
+		public override string Description
+		{
+			get
+			{
+				return "The EmitInfantryOnSell trait was renamed to SpawnActorsOnSell and the default\n" +
+					"actor type to spawn was removed. Uses of the old traits and defaults are updated\n" +
+					"to account for this.";
+			}
+		}
 
-        public override IEnumerable<string> UpdateActorNode(ModData modData, MiniYamlNode actorNode)
-        {
-            foreach (var uneios in actorNode.ChildrenMatching("-EmitInfantryOnSell"))
-                uneios.RenameKeyPreservingSuffix("-SpawnActorsOnSell");
+		public override IEnumerable<string> UpdateActorNode(ModData modData, MiniYamlNode actorNode)
+		{
+			actorNode.RenameChildrenMatching("-EmitInfantryOnSell", "-SpawnActorsOnSell");
```

Comment:
We are generally really bad about (not) renaming removals in most cases.  How about we add a `bool includeRemovals=true` parameter to the `*Rename*` helpers so that we don't need to worry about it?